lib/SIL/IR/SILModule.cpp

```diff
@@ -518,6 +518,17 @@ SILVTable *SILModule::lookUpVTable(const ClassDecl *C,
   if (!Vtbl)
     return nullptr;
 
+  if (C->walkSuperclasses([&](ClassDecl *S) {
+    SILVTable *Vtbl = getSILLoader()->lookupVTable(S);
+    if (!Vtbl) {
+      return TypeWalker::Action::Stop;
+    }
+    VTableMap[C] = Vtbl;
```

Comment:
I don't understand this. `VTableMap[C]` is overwritten for each superclass?

Reply:
Oops, this needs to be `VTableMap[S] = Vtbl;`, not `C`.